src/components/Verify.js

```diff
@@ -30,76 +30,84 @@ import FilterTop from './FilterTop';
-import { selectedHighlightColor, documentTitle } from '../common/variables.js';
-import { countToLocaleString } from '../common/numbers';
-import { VerifyContext } from '../context/VerifyContext';
-import { SpeciesContext } from '../context/SpeciesContext';
-import { TagsContext } from '../context/TagsContext';
-import { CaptureDetailProvider } from '../context/CaptureDetailContext';
-
-const log = require('loglevel').getLogger('../components/Verify');
+import { documentTitle } from 'common/variables';
+import { countToLocaleString } from 'common/numbers';
+import { VerifyContext } from 'context/VerifyContext';
+import { SpeciesContext } from 'context/SpeciesContext';
+import { TagsContext } from 'context/TagsContext';
+import { CaptureDetailProvider } from 'context/CaptureDetailContext';
+
+const log = require('loglevel').getLogger('components/Verify');
 const EMPTY_ARRAY = new Array(16).fill();
 const SIDE_PANEL_WIDTH = 315;
 
 const useStyles = makeStyles((theme) => ({
   wrapper: {
-    padding: theme.spacing(2, 8, 4, 8),
-  },
-  cardImg: {
-    width: '100%',
-    height: 'auto',
-  },
-  cardTitle: {
-    color: '#f00',
+    padding: theme.spacing(2, 8),
   },
   card: {
+    position: 'relative',
+    display: 'grid',
+    padding: 0,
     cursor: 'pointer',
+    borderRadius: '16px',
     '&:hover $cardMedia': {
       transform: 'scale(1.04)',
     },
+    '&:hover $cardActions': {
+      opacity: 1,
+      transition: theme.transitions.create('opacity', {
+        easing: theme.transitions.easing.easeInOut,
+        duration: '.8s',
+      }),
+    },
+    '&:hover $cardDetail': {
+      opacity: 1,
+      transition: theme.transitions.create('opacity', {
+        easing: theme.transitions.easing.easeInOut,
+        duration: '.8s',
+      }),
+    },
+  },
+  cardWrapper: {
+    padding: theme.spacing(2),
   },
   cardCheckbox: {
-    position: 'absolute',
+    gridArea: '1/-1',
     height: '1.2em',
     width: '1.2em',
-    top: '0.2rem',
-    left: '0.3rem',
+    margin: '4px',
     pointerEvents: 'none',
-    display: 'flex',
-    alignItems: 'center',
-    justifyContent: 'center',
+    borderRadius: '50%',
     zIndex: 1,
   },
   cardSelected: {
     backgroundColor: theme.palette.action.selected,
-  },
-  cardContent: {
-    padding: 'calc(400%/3) 0 0 0',
-    position: 'relative',
-    overflow: 'hidden',
-    display: 'flex',
-    alignItems: 'center',
-    justifyContent: 'center',
-  },
-  selected: {
-    border: `2px ${selectedHighlightColor} solid`,
+    borderRadius: '24px',
   },
   cardMedia: {
-    position: 'absolute',
-    top: 0,
-    bottom: 0,
-    left: 0,
-    right: 0,
+    gridArea: '1/-1',
     transform: 'scale(1)',
     transition: theme.transitions.create('transform', {
       easing: theme.transitions.easing.easeInOut,
       duration: '0.2s',
     }),
   },
-  cardWrapper: {
-    position: 'relative',
-    padding: theme.spacing(2),
-    flex: '1 0 45%',
+  cardActions: {
```

Comment:
I think these would be best aligned centrally, or spaced evenly across the card.